.github/workflows/h2-test.yml

```diff
@@ -19,9 +19,11 @@ jobs:
       matrix:
         include:
           - { java-version: 8,  scala-version: 2.12, sbt-opts: '' }
-          - { java-version: 8,  scala-version: 2.13,  sbt-opts: '' }
+          - { java-version: 8,  scala-version: 2.13, sbt-opts: '' }
+          - { java-version: 8,  scala-version: 3.3,  sbt-opts: '' }
           - { java-version: 11, scala-version: 2.12, sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
-          - { java-version: 11, scala-version: 2.13,  sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
+          - { java-version: 11, scala-version: 2.13, sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
+          - { java-version: 11, scala-version: 3.3,  sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
```

Comment:
Jvmcicompiler is removed in later version

Reply:
Thanks, I actually want to remove all of these experimental options but I will do it in a later PR.